src/components/Bookcard.jsx

```diff
@@ -24,10 +24,13 @@ const Bookcard = ({ book }) => {
           <button type="submit" className="bookbtn">Edit</button>
         </nav>
       </div>
-      <div className="progress">
-        <h3 className="percentage">{book.percent}</h3>
-        <p className="complete">Completed</p>
+      <div className="container">
+        <div className="oval" />
+        <div className="progress">
+          <h3 className="percentage">70%</h3>
+          <p className="complete">Completed</p>
```

Comment:
- [ ] One more thing, kindly ensure to give this element as pointed out earlier the correct `font-family` (Montserrat) according to the given template. 👍